index.html

```diff
@@ -1412,6 +1412,91 @@ <h3>Biometrics</h3>
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        Decentralized Identifiers may come to underpin much of our digital life. This 
+        may include public social and career personas, as well as more private personas 
+        shared among friends and family. Identifiers representing these personas and 
+        relationships may be some of the most important in our lives, so great care 
+        will be taken when choosing an identifier system to support these critical 
+        activities. As with all things, the appropriate and ethical use of technology
+        will be strongly considered  when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a
+        href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">
+        Principles</a> section of the Ethical Web Principles, there are key goals that
+        apply to all Web standards and technologies. DIDs are explicitly intended to
+        help users get closer to several of these goals, especially the following:
+        <ul>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">
+              Privacy
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">
+              Individual Control
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">
+              Device Independence
+            </a>
+          </li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Other concerns as applied to DID method development</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Considerations</h3>
+
+        <p>
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts caused by the method.
+        </p>
+
+        <p class="issue">
+          Some members of the working group feel it important that the [[DID-RUBRIC]] 
+          include environmental considerations as discussed in <a
+          href="https://github.com/w3c/did-rubric/pull/51">this pull request</a> in
+          order that users might appropriately evaluate and compare relevant aspects
+          of DID methods that might be applicable to their use cases.
+        </p>
+      </section>
```

Comment:
```suggestion
      <p>Many methods minimize environmental impact by leveraging existing infrastructure, such as
      the World Wide Web (with did:web) and Bitcoin (with did:btcr and did:ion). When considering any 
      particular approach, whether creating a new method or selecting among current methods, care 
      should be taken to  distinguish between the marginal impact of the additional use of such 
      infrastructure as a result of the  DID method and the existing impact from current use. For example,
      the did:web implementation is unlikely to have a significant additive environmental impact above
      and beyond the existing World Wide Web. 
      </p>
      </section>
```

Reply:
This wording acknowledges a common misperception and corrects the framing error that makes it so juicy from a memetic perspective, without presuming guilt or setting up any type of accusation in a place where the conversation would not afford the inferred respondents their due say.

I do not oppose the environmental concerns subsection, if it uses this wording.

I would still like the final version to include a link to relevant Rubric criteria.

Reply:
any concern with calling out specific methods? I have tried to avoid that to date, especially as methods may change over time

Reply:
I'm fine with it, https://html.spec.whatwg.org/multipage/system-state.html#safelisted-scheme already mentions bitcoin... as well as other things.

Reply:
+1 will commit suggestion